programs/mango-v4/src/health/cache.rs

```diff
@@ -717,9 +731,11 @@ pub fn new_health_cache(
         token_infos.push(TokenInfo {
             token_index: bank.token_index,
             maint_asset_weight: bank.maint_asset_weight,
-            init_asset_weight: bank.scaled_init_asset_weight(prices.asset(HealthType::Init)),
+            init_asset_weight: bank.init_asset_weight,
+            init_scaled_asset_weight: bank.scaled_init_asset_weight(prices.asset(HealthType::Init)),
```

Comment:
ok I just noticed this, why is deposit_weight_scale_start_quote compared with deposits computed using potentially stable price? hmm shouldn't it use raw oracle price always?

same for scaled_init_liab_weight below

Reply:
Mmh. Could argue that using the oracle price leads to stricter results for asset weight scaling (because the stable price can only reduce the combined asset price). For liab weight the combined price leads to stricter results because it can only be >= oracle.

I'll make the change.